### PR TITLE
Pull request for project review

### PR DESCRIPTION
**Pull request by leonmezu1**
# Project requirements:

 - [x]  Build a Rails application ensuring that:
    - The models, associations, controllers, and views are working as expected
    - Sign-up, log-in and log-out methods within the app sessions behavior work successfully
    - Models' special associations and scopes work properly
    - Users can create, view, edit and destroy events
    - Users can assist events

Further information is available at [The Odin Project Associations](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations)

### CODE REVIEW

**Comment by Usagib** [comment]
PR for code review: Lint errors fixed

File: app/views/events/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <h3><%= @event.location %></h3>
+      <h4><%= @event.description %></h4>
+
+      <% if @event.upcoming? %>
+        <% unless @event.attendees.include?(current_user) %>
+          <%= link_to  'Attend', schedule_event_event_path(@event), class: "btn btn-primary", method: :post%>
+        <% else %>
+          <%= link_to  'Cancel attendance', remove_event_event_path(@event), class: "btn btn-danger", method: :post%>
+        <% end %>
+      <% end %>
```
Comment (mosaaleb) [inline app/views/events/show.html.erb:22]: - [ ] The event's show page displays the list of attendants

File: app/views/events/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    </div>
+    <div class="col-lg-4">
+      <h3>created by: <%= @event.user.username %></h3>
+      <h1><%= @event.name %></h1>
+      <h3><%= @event.event_date %></h3>
+      <h3><%= @event.location %></h3>
+      <h4><%= @event.description %></h4>
+
+      <% if @event.upcoming? %>
```
Comment (mosaaleb) [inline app/views/events/show.html.erb]: - [ ] There shoud be absolutely NO logic in views.



Please move the logic to helpers.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Well done Team :clap:

The projects looks really nice, most requirements are handled, there is only two minor requirements to handle before moving on. Please take a look at my comments below.


#### Please handle these requirements and submit a new code review.

##### Muhammad Ebeid

**Comment by leonmezu1** [comment]
Required changes completed

**Review by nikasulo — APPROVED** [review]
# Great work! @leonmezu1 and @Usagib :medal_military: 
Please merge and keep moving forward! Great job! I wish you the very best and see you at the very top :rocket: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Pull request merged**